rtengine/iplab2rgb.cc

```diff
@@ -394,6 +430,138 @@ void ImProcFunctions::workingtrc(const Imagefloat* src, Imagefloat* dst, int cw,
         }
     };
 
+    if (profile == "sRGB" || profile == "Adobe RGB" || profile == "ProPhoto" || profile == "WideGamut" || profile == "BruceRGB" || profile == "Beta RGB" || profile == "BestRGB" || profile == "Rec2020" || profile == "ACESp0" || profile == "ACESp1") {
+        if (settings->verbose) {
+            printf("Profile=%s\n", profile.c_str());
+        }
+    } else {
+        if (settings->verbose) {
+            printf("profile not accepted\n");
+        }
+        return;
+    }
+
+    if(mul == -5 &&  gampos == 2.4 && slpos == 12.92310) {//must be change if we change settings RT sRGB
+        //only in this case we can shortcut..all process..no gamut control..because we reduce...leads to very small differences, but big speedup
+#ifdef _OPENMP
+            #pragma omp for schedule(dynamic, 16) nowait
```

Comment:
Shouldn't this be 
```
#pragma omp parallel for schedule(dynamic, 16) if (multiThread)
```
?

Reply:
Yes, you are true... I change.
Thank you  :)

Jacques